naas_python/domains/secret/adaptors/primary/utils.py

```diff
@@ -0,0 +1,40 @@
+from typing import List
```

Comment:
Is this file tied to Secrets only? It feels like it could be moved to the `utils` directory no? If we make it generic